core/src/main/java/org/apache/metamodel/query/SelectItem.java

```diff
@@ -152,7 +166,24 @@ public SelectItem(Column column, FromItem fromItem) {
      * @param fromItem
      */
     public SelectItem(FunctionType function, Column column, FromItem fromItem) {
-        this(column, fromItem, function, null, null, null, false);
+        this(column, fromItem, function, null, null, null, null, false);
```

Comment:
Would it make more sense to put the if before the creation of the object? I mean if the column is null and we're throwing an exception, why are we instantiating the object?